### PR TITLE
[Website] : Add FAQ section

### PR DESCRIPTION
**Pull request by zekth**
As discussed with @ry and @bartlomieju , here is a draft of the FAQ section. Feel free to contribute to it.

### CODE REVIEW

**Comment by ry** [comment]
I’d rather not bifurcate the documentation - can these be worked into the manual somehow?

**Comment by zekth** [comment]
> I’d rather not bifurcate the documentation - can these be worked into the manual somehow?

Totally. Adding it following `Contributing` section?

**Comment by ry** [comment]
@zekth Or an FAQ section would make sense too. As per usual, I think we should use Go as an example of how to structure things: https://golang.org/doc/

**Comment by zekth** [comment]
Updated. I think people may have more FAQ items to add.

**Comment by bartlomieju** [comment]
@zekth, I still believe these questions are most common ones:
> What is Deno?
How is Deno different that node.js?
Should I switch from node to Deno?

File: website/manual.md [hunk 1]
```diff
@@ -784,6 +784,34 @@ These Deno logos, like the Deno software, are distributed under the MIT license
 
 - [A pixelated animation one by @tanakaworld](https://github.com/denoland/deno/blob/master/website/images/deno_logo_4.gif)
 
+## Frequently Asked Questions
+
+re is the FAQ section, where you'll maybe find the answers of your questions.
```
Comment (bartlomieju) [inline website/manual.md]: Typo

File: website/manual.md [hunk 2]
```diff
@@ -784,6 +784,43 @@ These Deno logos, like the Deno software, are distributed under the MIT license
...
+### How Deno is different from node?
+
+- Deno is secure by default
+- Deno only supports ES modules
+- Deno supports TypeScript natively
+- Deno is only one binary
+
+### Is deno going to support node?
+
+No, but there is some polyfills in development.
```
Comment (ry) [inline website/manual.md]: Add link

File: website/manual.md [hunk 3]
```diff
@@ -784,6 +784,43 @@ These Deno logos, like the Deno software, are distributed under the MIT license
...
+
+### Should i switch from node to Deno?
```
Comment (ry) [inline website/manual.md]: "i" should be upper case.

File: website/manual.md [hunk 4]
```diff
@@ -784,6 +784,43 @@ These Deno logos, like the Deno software, are distributed under the MIT license
...
+
+No, as it's not production ready you have no benefit to do the switch.
```
Comment (ry) [inline website/manual.md]: Link to https://deno.land/manual.html#disclaimer

File: website/manual.md [hunk 5]
```diff
@@ -784,6 +784,43 @@ These Deno logos, like the Deno software, are distributed under the MIT license
...
+
+First of all thanks for the help. You may first start to parse all the issues in
+[deno](https://github.com/denoland/deno) and
+[deno_std](https://github.com/denoland/deno) repositories. If you want to add
+functionnalities, be sure to start an issue to discuss about it. Also please
+refer to the [contributing section](#Contributing).
+
+### How Deno is different from node?
+
+- Deno is secure by default
```
Comment (ry) [inline website/manual.md]: Can you expand on this? What about the language it's written in? What about resources/ops? What about libuv/tokio?  GN vs GYP? production ready vs experimental?

File: website/manual.md [hunk 6]
```diff
@@ -784,6 +784,43 @@ These Deno logos, like the Deno software, are distributed under the MIT license
...
+
+No.
+
+### Will there be an official package manager for Deno?
+
+No.
+
+### How came the Deno name?
+
+Just swap the letters and `node` become `deno`.
```
Comment (ry) [inline website/manual.md]: Remove this. It's obvious and not practical information. It doesn't need to be said.

File: website/manual.md [hunk 7]
```diff
@@ -784,6 +784,43 @@ These Deno logos, like the Deno software, are distributed under the MIT license
...
 
+## Frequently Asked Questions
+
+### Is Deno production ready?
+
+No.
+
+### Will there be an official package manager for Deno?
+
+No.
```
Comment (ry) [inline website/manual.md]: Expand on this or remove.

File: website/manual.md [hunk 8]
```diff
@@ -784,6 +784,43 @@ These Deno logos, like the Deno software, are distributed under the MIT license
 
 - [A pixelated animation one by @tanakaworld](https://github.com/denoland/deno/blob/master/website/images/deno_logo_4.gif)
 
+## Frequently Asked Questions
+
+### Is Deno production ready?
+
+No.
```
Comment (ry) [inline website/manual.md]: Wasn't this covered in the last question? Remove.

File: website/manual.md (hunk 7 above)
Comment (zekth) [reply website/manual.md]: You want to add the example of the issues with `npm` and the fact we don't really know what we download?

File: website/manual.md (hunk 5 above)
Comment (zekth) [reply website/manual.md]: Ok, i'll expand this question. It would need some review for sure.

**Comment by ry** [comment]
@zekth The patch as it currently is doesn't add much information. In general I want to keep all the non-jsdoc documentation in the manual. It's not clear to me that we need a specific section called "FAQ".

**Pull request closed without merging**